src/test/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementControllerTest.java

```diff
@@ -11,16 +12,26 @@
 import uk.gov.companieshouse.api.psc.Statement;
 import uk.gov.companieshouse.pscstatementdataapi.services.PscStatementService;
 
+import javax.naming.ServiceUnavailableException;
 import java.time.LocalDate;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
```

Comment:
Could this .* be replaced with the imports as required?

Reply:
Sure can